utils/spacewalk-common-channels.ini

```diff
@@ -1328,6 +1328,38 @@ gpgkey_id = %(_uyuni_gpgkey_id)s
 gpgkey_fingerprint = %(_uyuni_gpgkey_fingerprint)s
 repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master:/openSUSE_Leap_15-Uyuni-Client-Tools/openSUSE_Leap_15.0/
 
+[opensuse_leap16_0]
+checksum = sha256
+archs    = x86_64, aarch64, s390x, ppc6le
+name     = openSUSE Leap 16.0 (%(arch)s)
+gpgkey_url = file:///usr/lib/rpm/gnupg/keys/gpg-pubkey-29b700a4-62b07e22.asc
+gpgkey_id = 29B700A4
+gpgkey_fingerprint = AD48 5664 E901 B867 051A B15F 35A2 F86E 29B7 00A4
+repo_url = https://cdn.opensuse.org/distribution/leap/16.0/repo/oss/
```

Comment:
While this repo was available at the instance as both `download.opensuse.org` and `cdn.opensuse.org`, it seems `cdn.opensuse.org` is the  future: https://news.opensuse.org/2023/07/31/try-out-cdn-with-opensuse-repos/

This change is [already part of openSUSE Leap Micro 6.0](https://news.opensuse.org/2024/11/20/leap-micro-61-alpha-availability/), so expect it to be part of openSUSE Leap 16.0